pkg/cluster/cluster.go

```diff
@@ -133,6 +134,10 @@ func new(config Config, e *spec.EtcdCluster, stopC <-chan struct{}, wg *sync.Wai
 		if err := c.createClientServiceLB(); err != nil {
 			return nil, fmt.Errorf("fail to create client service LB: %v", err)
 		}
+	} else {
+		if !c.bm.isBackupSetup() {
+			return nil, errors.New("")
```

Comment:
also it is still not clear to me that why do we need to check backup service if it is not a new cluster.

for example, if there is no backup policy given, do we still need to check this?